etc/kayobe/kolla/config/prometheus/prometheus.yml.d/10-redfish.yml

```diff
@@ -0,0 +1,24 @@
+{% if groups.get('redfish') %}
+scrape_configs:
+  - job_name: redfish-exporter
+    metrics_path: /redfish
+    scrape_timeout: 30s
+    scrape_interval: 5m
+    relabel_configs:
+     - source_labels: [__address__]
+       target_label: __param_target
+     - source_labels: [__param_target]
+       target_label: instance
+     - target_label: __address__
+       ### the address of the redfish-exporter address
+       replacement: "{{ groups['monitoring'][0] }}:9610"
+    static_configs:
+{% for host in groups.get('redfish', []) %}
+      - targets:
+          - '{{ hostvars[host]["ipmi_address"] }}'
+        labels:
+          server: '{{ host }}'
+          ### usually the rack or similar
+          env: "preprod"
```

Comment:
There are a few variables like this. I wonder if they could be encoded via a variable, so that you could keep the config files identical.

Reply:
Been thinking the same. Pierre actually adds one, which we can use, so maybe something for me to do after we merge in pierre's work.